variables/colors.scss

```diff
@@ -178,46 +178,52 @@ $colors: (
 $data-colors: (
   'data-navy': (
     color: #24598C,
-    contrast: #FFF,
+    contrast: var(--hdd-color-white),
     dark-theme: hsla(209, 100%, 35%, 1)
   ),
   'data-marine': (
     color: #116FCA,
-    contrast: #FFF,
+    contrast: var(--hdd-color-white),
     dark-theme: hsla(210, 100%, 43%, 1)
   ),
   'data-blue': (
     color: #4CA8FF,
-    contrast: #000,
+    contrast: var(--hdd-color-black),
     dark-theme: hsla(209, 100%, 71%, 1)
   ),
   'data-turquoise': (
     color: #25D2EF,
-    contrast: #000,
+    contrast: var(--hdd-color-black),
     dark-theme: hsla(189, 100%, 54%, 1)
   ),
   'data-seagreen': (
     color: #46B5AB,
-    contrast: #000,
+    contrast: var(--hdd-color-black),
     dark-theme: hsla(175, 77%, 49%, 1)
   ),
   'data-green': (
     color: #70E9AA,
-    contrast: #000,
+    contrast: var(--hdd-color-black),
     dark-theme: hsla(149, 85%, 68%, 1)
   ),
   'data-orange': (
     color: #F1832D,
-    contrast: #000,
+    contrast: var(--hdd-color-black),
     dark-theme: hsla(26, 98%, 56%, 1)
   ),
   'data-yellow': (
     color: #FFD20D,
-    contrast: #000,
+    contrast: var(--hdd-color-black),
     dark-theme: hsla(49, 100%, 62%, 1)
   )
 );
 
+
+// Dark-theme colors
+:root {
+  --hdd-color-grey-dark4: #333333;
+}
+
```

Comment:
hvorfor helt alene?

Reply:
Er eneste fargen som er bare brukt i dark-theme

Reply:
Har du noe bedre løsning på hvor den burde ligge?

Reply:
legg den i en @include is-dark-theme() {